webvtt/rendering/cues-with-video/processing-model/support/bidi_vertical_rl.vtt

```diff
@@ -0,0 +1,4 @@
+WEBVTT
+
+00:00:00.000 --> 00:00:05.000 vertical:rl
+<ruby>右<rt>みぎ</rt></ruby>に<ruby> 見<rt>み</rt></ruby>えるのは...
```

Comment:
Shouldn't this test produce more than one line so that the direction of layout of lines can be verified?

Reply:
Ihre Line is enough to distinguish vertical from horizontal I would think.